src/main/webapp/app/shared/components/documentation-button/documentation-button.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input } from '@angular/core';
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons';
+
+// The routes here are used to build the link to the documentation.
+// Therefore, it's important that they exactly match the url to the subpage of the documentation.
+// Additionally, the case names must match the keys in documentationLinks.json for the tooltip.
+export enum DocumentationType {
+    Course = <any>'courses/customizable/',
```

Comment:
Are these <any> necessary?

Reply:
Yes, without them we couldn't reverse map the type to its case name. We need the name for the translation string